testing/kuttl/e2e/pgadmin/02--check-settings.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # Log the amount of space on the startup volume.
```

Comment:
We don't want to compare this to what it should be, just log it for the user?

Reply:
🤔 My concern is that `sizeLimit: 32Ki` is too small despite my understanding. The result looks like this:

```
Filesystem     1K-blocks  Used Available Use% Mounted on
tmpfs                 32     4        28  13% /etc/pgadmin
```

Reply:
After some more testing, I'm confident we should expect 4KiB. Added an assertion.